src/index.js

```diff
@@ -2,34 +2,25 @@ import './style.css';
 import '@fortawesome/fontawesome-free/js/all.js';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
-const tasksDiv = document.getElementById('tasks');
-const tasks = [];
-class Task {
-  constructor(index, description, completed) {
-    this.index = index;
-    this.description = description;
-    this.completed = completed;
-  }
-}
-const task1 = new Task(1, 'first task', false);
-const task2 = new Task(2, 'second task', false);
-const task3 = new Task(3, 'third task', true);
-const task4 = new Task(4, 'forth task', true);
-tasks.push(task1, task2, task3, task4);
```

Comment:
- [ ] As you have deleted the default task, I wasn't able to check the drag and drop and completed functionality. Please add some task and also check all the functionality is working properly or not.

![Screenshot from 2021-07-22 04-01-19](https://user-images.githubusercontent.com/22434660/126568548-6c66b261-7c57-42fa-aea5-d61ddf6b5289.png)